plugin/src/main/java/io/jenkins/plugins/casc/impl/secrets/VaultSecretSource.java

```diff
@@ -110,4 +117,13 @@ private String getVariable(String key, Properties prop) {
             return System.getenv(key);
         }
     }
+
+    private String[] getCommaSeparatedVariables(String key, Properties prop) {
+        if (prop != null && !prop.isEmpty()) {
```

Comment:
This should properly just call getVariable and then split :|

Reply:
You are right. Changed it to call `getVariable`.